internal/engine/actions/alert/alert.go

```diff
@@ -48,6 +49,18 @@ func NewRuleAlert(
 		}
 		return security_advisory.NewSecurityAdvisoryAlert(
 			ActionType, ruletype, alertCfg.GetSecurityAdvisory(), client, setting)
+	case pull_request_comment.AlertType:
```

Comment:
Do we want to feature gate this?

Reply:
I was going to leave it undocumented until it's ready, but if feature gating is the right approach then we can do that.

Reply:
After thinking about it a bit longer, I will add a feature gate, that seems like the right thing

Reply:
I don't mind doing it, but ss it really necessary to feature flag this one?